cdk-testsuite/tests/vagrant_sanity.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/python
+
+from avocado import Test
+from avocado.utils import process
+import os, re, time, pexpect, vagrant
+
+class VagrantSanity(Test):
+    def setUp(self):
+	self.vagrant_VAGRANTFILE_DIR = self.params.get('vagrant_VAGRANTFILE_DIR')
+        self.vagrant_PROVIDER = self.params.get('vagrant_PROVIDER', default='')
+	self.vagrant_RHN_USERNAME = self.params.get('vagrant_RHN_USERNAME')
+	self.vagrant_RHN_PASSWORD = self.params.get('vagrant_RHN_PASSWORD')
+	os.chdir(self.vagrant_VAGRANTFILE_DIR)
```

Comment:
This can fail, for example when path is not defined or incorrect, exceptions should be handled here. Also it could be also checked if there is Vagrantfile in directory.
If any of these check fail, setUp() should probably fail too, what do you think?